virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc

```diff
@@ -0,0 +1,15 @@
+[id="virt-configuring-a-watchdog"]
+= Configuring a watchdog
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-a-watchdog
+
+toc::[]
+
+Expose a watchdog device on a virtual machine instance (VMI), alongside liveness and readiness probes, to ensure the operating system is running properly.
```

Comment:
I noticed that Daniel K asked you to use a VM CR  instead of a VMI CR. So virtual machine instance (VMI) is no longer relevant.